packages/shorebird_cli/lib/src/extensions/iterable.dart

```diff
@@ -0,0 +1,5 @@
+/// Provides the [containsAnyOf] method to all [Iterable]s.
+extension ContainsAnyOf<T> on Iterable<T> {
+  /// Returns `true` if any of the elements in [elements] are in this iterable.
+  bool containsAnyOf(Iterable<T> elements) => elements.any(contains);
```

Comment:
It looks a bit nicer with list literals:

```dart
argResults.options.containsAnyOf([a,b]);
```

instead of 

```dart
[a,b].any((e) => argResults.options.contains(e));
```

Mostly just a minor readability improvement, happy to remove if you feel strongly.